technical-reports/format/design-token.md

```diff
@@ -23,6 +23,14 @@ The example above therefore defines 1 design token with the following properties
 
 Name and value are both **required**.
 
+### Character restrictions
+
+Due to the syntax used for [token aliases](#aliases-references) the following characters cannot be used in a token's **name** property or in a [token group](#groups-0) name:
+
+- `{`
+- `}`
+- `.`
```

Comment:
Would it be helpful to additionally write out the names of the characters? E.g.:

```suggestion
- `{` (left curly bracket)
- `}` (right curly bracket)
- `.` (period)
```